renderdoc/driver/gl/gl_common.h

```diff
@@ -71,10 +77,17 @@ struct GLWindowingData
     wnd = 0;
   }
 
+#if defined(RENDERDOC_SUPPORT_GL)
```

Comment:
Likewise here, we'll need to have the data for both - maybe under a `struct GL` and `struct EGL` ? The `SetCtx` function will maybe need to set ctx in both, but that's fairly harmless. I think the rest of the code will know if it's dealing with GL or EGL, so it can use the appropriate members.